### PR TITLE
change query schema editor to monaco-editor

### PR DESCRIPTION
**Pull request by hyzx86**
image

### CODE REVIEW

**Comment by hyzx86** [comment]
However, the monaco editor still does not work after I have commented all the references to the codemirror..

**Comment by hyzx86** [comment]
@hishamco @Skrypt Can you help me look at this problem?

**Comment by Skrypt** [comment]
> No adjustment can make the editor effective on this page

Can we see a screencast of the actual issue to define what it is?

**Comment by hyzx86** [comment]
https://user-images.githubusercontent.com/15613121/191302632-f79c9c22-3fd4-4abe-8810-0a232fd6801d.mp4

**Comment by hyzx86** [comment]
This is a clearer version
https://user-images.githubusercontent.com/15613121/191303275-9b922f85-d421-4a22-8c34-fc57b655daa0.mp4

**Comment by hyzx86** [comment]
🤣Done ， I have to change the editor container to div instead the textarea

image

File: src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml:1]: Remove extra line

File: src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml [hunk 2]
```diff
@@ -10,6 +12,42 @@
 
 <div class="mb-3">
     <label asp-for="Schema">@T["Schema"]</label>
-    <textarea asp-for="Schema" rows="12" class="form-control" style="text-align:left"></textarea>
+    <textarea asp-for="Schema" hidden rows="12" class="form-control" style="text-align:left"></textarea>
+    <div id="@Html.IdFor(x=>x.Schema)_editor" style="min-height: 300px;" class="form-control"></div>
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml:16]: Html.IdFor(x => x.Schema)

**Review by hishamco — CHANGES_REQUESTED** [review]
Please check the format in every PR, also share a screen cast as @Skrypt suggested

**Review by sebastienros — APPROVED** [review]

**Pull request merged**